ajax/libs/libphonenumber/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "libphonenumber",
+  "filename": "phonenumberutil.min.js",
+  "version": "8.0.0",
+  "description": "Google's common Java, C++ and JavaScript library for parsing, formatting, and validating international phone numbers.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googlei18n/libphonenumber.git"
+  },
+  "keywords": [
+    "google",
+    "phonenumber"
+  ],
+  "author": "googlei18n <https://developers.google.com/international>",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/googlei18n/libphonenumber.git",
+    "fileMap": [
+      {
+        "basePath": "javascript/i18n/phonenumbers",
+        "files": [
+          "asyoutypeformatter!(_test*)",
+          "metadata!(fortesting*)",
+          "phone!(*_test*)"
```

Comment:
It's different case. BTW, `!(test*)` is wrong rule, but I think `!(*test*)` it's ok.